doc/usage/configuration.rst

```diff
@@ -2486,6 +2486,13 @@ These options influence Texinfo output.
 
    .. versionadded:: 1.1
 
+.. confval:: texinfo_emit_document_references
```

Comment:
Is "document references" a commonly used word in texinfo? It's the first time I heard it. If not, it would be better to use "cross-references" instead. It's a common word of Sphinx. How about `texinfo_cross_references = True | False`?